main.go

```diff
@@ -139,12 +139,26 @@ func getConnWithRetry(flags *flagStorage) (c *gcsx.Connection, err error) {
 ////////////////////////////////////////////////////////////////////////
 
 // Mount the file system according to arguments in the supplied context.
+
+func getUserAgent(AppName string) string {
+	var userAgent string
+
+	if os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE") != "" {
```

Comment:
Do you also think we can simplify it by setting user agent like:
userAgent = strings.TrimSpace(fmt.Sprintf("gcsfuse/%s %s %s", getVersion(), appName, os.Getenv("GCSFUSE_METADATA_IMAGE_TYPE")))

Reply:
fixed